plugins/io

```diff
@@ -0,0 +1 @@
+repository = https://github.com/mracos/asdf-io
```

Comment:
I think the actual Git repository name ends in `.git`. Sorry I just merged the PR, can you test this and create another one if this does not work?

Reply:
Will look into it :smile: